server/src/main/java/org/elasticsearch/ElasticsearchException.java

```diff
@@ -1047,7 +1047,12 @@ public String toString() {
                 org.elasticsearch.transport.NoSeedNodeLeftException.class,
                 org.elasticsearch.transport.NoSeedNodeLeftException::new,
                 160,
-                Version.V_7_10_0);
+                Version.V_7_10_0),
+        VERSION_MISMATCH_EXCEPTION(
+                org.elasticsearch.action.search.VersionMismatchException.class,
+                org.elasticsearch.action.search.VersionMismatchException::new,
+                161,
+                Version.V_7_11_0);
```

Comment:
For now it's just a 8.0.0 exception, you'll need to adapt the version **after** the backport